test/unit_tests/datautil/test_windowers.py

```diff
@@ -14,13 +16,15 @@
 from braindecode.datasets.moabb import fetch_data_with_moabb
 from braindecode.datautil import (
     create_windows_from_events, create_fixed_length_windows)
+from braindecode.datautil.preprocess import MNEPreproc, preprocess
```

Comment:
from xxx.preprocess import preprocess 

I don't like that as it's then confusing what is a function and what is a module

Reply:
What about we rename `braindecode.datautil.preprocess` to `braindecode.datautil.preprocessing`? It's a bit longer, but at least we avoid this ambiguity. @robintibor @gemeinl

Reply:
Personally, I don't like the name "datautil" at all and feel like we could use some major refactoring here. My thoughts:
Currently, ``braindecode.datautil`` for me contains a mess of 
- Loading and saving datasets (``serialization.py``) 
- Creating datasets (``xy.py``, ``mne.py``)
- Preprocessing (``preprocess.py``)
- Creating windows ((``windowers.py``) which is also a bad name, since we don't have windower classes anymore). Also, creating windows is a kind of preprocessing

We could probably rename ``braindecode.datautil`` to ``braindecode.preprocessing`` with ``preprocessing.py`` which contains a merge of ``preprocess.py`` and ``windowers.py``.

We could add ``braindecode.serialization`` with ``serialization.py``

I am not so sure what to do about ``xy.py`` and ``mne.py``, though.